repos/system_upgrade/el7toel8/actors/checkkdeapps/libraries/library.py

```diff
@@ -0,0 +1,25 @@
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM
+
+
+def get_kde_apps_info():
+    installed = list()
+    base_kde_apps = ("kde-baseapps",
+                     "okular",
+                     "ark",
+                     "kdepim",
+                     "konsole",
+                     "gwenview",
+                     "kdenetwork",
+                     "kate",
+                     "kwrite")
```

Comment:
@Honza0297, is it OK if the upgrade passes just fine with message in the report when I have `kate-part` and `kde-runtime` installed?

Reply:
There is a question what is the "base" application. The idea is to check only these apps, because they should be the "most used" ones.